explore/precision.py

```diff
@@ -196,6 +206,23 @@ class Kernel(object):
     model_info = modelinfo.make_model_info(Kernel)
     return model_info
 
+# Hack to allow second parameter A in two parameter functions
```

Comment:
This hack/method smells from a mile away but since it's just a quick testing program, it causes no trouble...

Reply:
Agreed.  I added more comments in the code.